nucleus/glassfish-jul-extension/src/main/java/org/glassfish/main/jul/cfg/ConfigurationHelper.java

```diff
@@ -85,7 +88,7 @@ public class ConfigurationHelper {
         if (v == null || v.isEmpty()) {
             return Collections.emptyList();
         }
-        return Arrays.asList(v.split("[\\s]*,[\\s]*"));
+        return Arrays.asList(PATTERN_STR_TO_LIST.split(v, 0));
```

Comment:
If the pattern matching only means split and trim, I thought it would be better to execute it clearly.
such as...
```java
return Arrays.stream(v.split(",")).map(String::trim).collect(Collectors.toList());
```

Reply:
I think that this would be much slower - it creates a stream object, parses the pattern for every value, then calls trim function for every result, and creates an array list.
While my version uses pregenerated Pattern and creates an ArrayList directly backed by the result array.

Reply:
But the truth is that for several values from a property file it doesn't matter.

Reply:
I have updated just the commit with your recommendation, thank you!